drivers/graphtest/graphtest_unix.go

```diff
@@ -312,27 +313,42 @@ func writeRandomFile(path string, size uint64) error {
 }
 
 // DriverTestSetQuota Create a driver and test setting quota.
-func DriverTestSetQuota(t *testing.T, drivername string) {
+func DriverTestSetQuota(t *testing.T, drivername string, required bool) {
```

Comment:
This is only being called with `required` set to `false`.  Is it supposed to be called with `required` set to `true` somewhere?

Reply:
Moby has an update to zfs that calls it with true, I updated the zfs driver to show the use case.